core/cfdefs/cfdef_standard.php

```diff
@@ -280,9 +280,10 @@ function cfdef_prepare_date_value( $p_value ) {
  * print_custom_field_input
  * @param array $p_field_def          Custom field definition.
  * @param mixed $p_custom_field_value Custom field value.
+ * @param string $p_required          The "required" attribute to add to the field
  * @return void
  */
-function cfdef_input_list( array $p_field_def, $p_custom_field_value ) {
+function cfdef_input_list( array $p_field_def, $p_custom_field_value, $p_required = '' ) {
```

Comment:
Should the parameter $p_required be a true/false flag?, like for example, `print_custom_field_input`

Reply:
I found it easier to directly pass the HTML attribute rather than a boolean flag, all in all that seemed less error-prone and the code is structured more clearly.